doc/conf.py

```diff
@@ -35,13 +36,15 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
-    'numpydoc',
+    # 'numpydoc',
     'sphinx.ext.viewcode',
     'sphinx.ext.doctest',
     'sphinx.ext.intersphinx',
     'sphinx.ext.imgconverter',
-    # 'sphinx_gallery.gen_gallery',
+    'sphinx_gallery.gen_gallery',
     'sphinx_issues',
+    'sphinx_rtd_theme',
+    'sphinx.ext.napoleon'
```

Comment:
Numpydoc vs sphinx.ext.napoleon is a choice: both are possible.

Reply:
So, actually, argument names and parameters were not parsed properly with `numpydoc`, that's why i switched. Perhaps it can be configured, but this was easier.